src/test/kotlin/org/rustSlowTests/cargo/runconfig/RunConfigurationTest.kt

```diff
@@ -214,4 +214,29 @@ class RunConfigurationTest : RunConfigurationTestBase() {
         check("2. bbb" in stdout)
         check("3. ccc" !in stdout)
     }
+
+    fun `test toolchain override`() {
+        fileTree {
+            toml("Cargo.toml", """
+                [package]
+                name = "hello"
+                version = "0.1.0"
+                authors = []
+            """)
+
+            dir("src") {
+                rust("main.rs", """
+                    #![feature(if_let)]
+
+                    fn main() {
+                        println!("Hello, world!");
+                    }
+                """)
+            }
+        }.create()
+        val configuration = createConfiguration("+nightly run")
```

Comment:
Looks like it's not a correct way to pass `+nightly` in this way because the current implementation puts it into `command` field of configuration

Reply:
Nevermind
Everything is fine
Sorry for confusion